templates/download/xilinx/index.html

```diff
@@ -31,78 +31,293 @@ <h1>Install Ubuntu on Xilinx</h1>
   </div>
 </section>
 
-<section class="p-strip--light">
-  <div class="row u-vertically-center">
-    <div class="col-6">
-      <h2>Download Ubuntu Desktop</h2>
-      <h3 class="p-heading--4">Ubuntu Desktop {{ releases.lts.full_version }} LTS</h3>
-      <p class="u-sv1">The version of Ubuntu with up to 10 years of long term support, until April 2030.</p>
-      <p>
-        <a class="p-button--positive" href="https://people.canonical.com/~platform/images/xilinx/iot-zcu10x-classic-desktop-2004-x07-20210728-85.img.xz">Download 64-bit</a>
-      </p>
-      <p>Works on:</p>
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">Xilinx ZCU102</li>
-        <li class="p-list__item is-ticked">Xilinx ZCU104</li>
-        <li class="p-list__item is-ticked">Xilinx ZCU106</li>
-        <li class="p-list__item is-ticked">Xilinx Kria KV260 Vision AI Starter Kit&nbsp;&nbsp;<div class="p-label">Coming soon</div></li>
-      </ul>
-    </div>
-    <div class="col-6 u-align--center u-hide--small">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/24e23fbe-zcu106-evaluation-board.png",
-        alt="",
-        width="400",
-        height="273",
-        hi_def=True,
-        loading="auto"
-        ) | safe
-      }}
-    </div>
-  </div>
-</section>
-
 <section class="p-strip">
```

Comment:
There's a lot of whitespace between the hero strip and the tabbed content, what about:
```suggestion
<section class="p-strip is-shallow">
```

Reply:
Thanks for the suggestions, definitely looks a lot cleaner now